### PR TITLE
Remove references to Photon Controller from vDVS docs [SKIP CI]

### PR DESCRIPTION
**Pull request by shaominchen**
Fixes issue #1647: Remove references to Photon Controller from vDVS docs

### CODE REVIEW

File: docs/user-guide/docker-plugin-drivers.md [hunk 1]
```diff
@@ -3,31 +3,16 @@ title: Overview
 
 ---
 ## Overview
-The Docker volume plugin supports the below platforms and the corresponding drivers for those. The plugin supports all volume provisioning and managenment operations, defined by the Docker Volume plugin interface, on both platforms. Both drivers allow users to provision and use VMDK backed volumes for containers in Docker.
-
-1. vSphere
-2. Photon
-
-<script type="text/javascript" src="https://asciinema.org/a/80417.js" id="asciicast-80417" async></script>
+The Docker volume plugin supports the vSphere platform and provides corresponding driver for it. The plugin supports all volume provisioning and managenment operations, defined by the Docker Volume plugin interface. The corresponding driver allow users to provision and use VMDK backed volumes for containers in Docker.
```
Comment (shuklanirdesh82) [inline docs/user-guide/docker-plugin-drivers.md]: nit: "The corresponding driver allow" => "The corresponding driver allows"

File: docs/user-guide/docker-plugin-drivers.md [hunk 2]
```diff
@@ -38,19 +23,10 @@ The docker volume plugin supports two drivers, namely, `photon` and `vsphere` fo
 ## Sample plugin configuration
 ```
 {
-	"Driver": "<driver name - vsphere/vmdk/photon>",
+	"Driver": "<driver name - vsphere>",
```
Comment (shuklanirdesh82) [inline docs/user-guide/docker-plugin-drivers.md]: how about this? `"Driver": "vsphere",`

File: docs/user-guide/docker-plugin-drivers.md (hunk 1 above)
Comment (shaominchen) [reply docs/user-guide/docker-plugin-drivers.md]: Done.

File: docs/user-guide/docker-plugin-drivers.md (hunk 2 above)
Comment (shaominchen) [reply docs/user-guide/docker-plugin-drivers.md]: Done.

**Pull request merged**